manifests/apache_exporter.pp

```diff
@@ -89,6 +89,7 @@
   Boolean $restart_on_change                                         = true,
   Boolean $service_enable                                            = true,
   String[1] $service_ensure                                          = 'running',
+  String $service_name                                               = 'apache_exporter',
```

Comment:
I like this change, but there are a lot of exporters which still use a static service_name:
```
apache_exporter.pp
beanstalkd_exporter.pp
collectd_exporter.pp
consul_exporter.pp
elasticsearch_exporter.pp
graphite_exporter.pp
haproxy_exporter.pp
mesos_exporter.pp
mongodb_exporter.pp
mysqld_exporter.pp
nginx_vts_exporter.pp
process_exporter.pp
rabbitmq_exporter.pp
server.pp
snmp_exporter.pp
statsd_exporter.pp
varnish_exporter.pp
```

Reply:
are you suggesting that this change should also make the `service_name` dynamic in all those? that change is probably part of #303 anyways...